src/nbl/video/utilities/IUtilities.cpp

```diff
@@ -76,24 +76,21 @@ void IUtilities::updateImageViaStagingBuffer(
         {
             // but first sumbit the already buffered up copies
             cmdbuf->end();
-            IGPUQueue::SSubmitInfo submit;
+            IGPUQueue::SSubmitInfo submit = submitInfoBeforeOverwrite;
             submit.commandBufferCount = 1u;
             submit.commandBuffers = &cmdbuf;
```

Comment:
you do not want without changing the commandbuffer count and commandbuffers to be submitted, because now you take a whole `intendedSubmitOfPreceedingWork`